lib/importproject.h

```diff
@@ -100,9 +100,18 @@ class CPPCHECKLIB WARN_UNUSED ImportProject {
     bool importCompileCommands(std::istream &istr);
     bool importCppcheckGuiProject(std::istream &istr, Settings *settings);
     virtual bool sourceFileExists(const std::string &file);
+
 private:
+    struct SharedItemsProject {
+        bool successFull;
```

Comment:
nit I don't think "success" and "full" should be separated it's 1 word.

Reply:
Fixed typo / whatever that was in e1c2280